CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 # Changelog
 
+## 1.1.2
```

Comment:
@kchan-varicent can we just use this version #?  or you want a new version number?  ours is like 0.3.8

Reply:
maybe 1.1.3? it really doesn't matter as long as we're increasing.